about.html

```diff
@@ -25,7 +25,7 @@
             </nav>
         </header>
         <div class="top-content">
-            <img src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png">
+            <img class="image" src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png">
```

Comment:
Okay, so I'm not seeing these images. The path you used to call them might work on your computer locally, but doesn't work for project up on Github. Rather than `src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png"` I think we need something like `src="img\jumbo-about.png"`. That will call it in this directory. It basically looks in the `img` folder and then in this case 
for the `jumbo-about.png` file

Reply:
src="img\jumbo-about.png"